.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,11 +8,10 @@
 [ ] Add a CLI option
 [ ] Add something to the core
 [ ] Other, please explain:
-```
-
-*If the item you've check above has a template, please paste the template questions here and answer them.*
-
 
+<!--
+    If the item you've check above has a template, please paste the template questions here and answer them. (If this pull request is addressing an issue, you can just paste a link to the issue here instead.)
```

Comment:
Should we say "please paste the template questions **below**" and "paste a link to the issue **below**" instead? (Emphasis mine.) Reason is in case someone puts the questions inside this HTML comment, which would render them invisible when processed by the Markdown engine.

Hopefully we don't have too many users who would do that, but still.